client/src/Pages/PlayerWaitingScreen.jsx

```diff
@@ -1,11 +1,129 @@
 import React from 'react';
-
+import {Button, Row, Col, Modal} from 'antd';
+import "../Styles/Waiting.scss";
+import RulesPage from './RulesPage';
+import CharcterPage from './CharacterPage';
 export default class PlayerWaitingScreen extends React.Component{
+    
+    state = {
+        rulesVisible: false,
+        charactersVisible: false,
+    };
+
+    showRules = () => {
```

Comment:
In a class, I've never seen methods declared like this. It should look like this:
```javascript
showRules() {
  ...
}
```
Look at the "Prototype methods" section [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes).

This happens a lot in this file.

Reply:
I'm Leaving the arrow functions for now, otherwise it breaks functionality, just so we can keep moving.

Reply:
Okay. FYI, I'm pretty sure the reason it doesn't work is because you aren't binding `this` to the functions. Check [this page](https://reactjs.org/docs/handling-events.html) out where they do exactly that. It's a thing I actually ran into on the server recently!

But if this works for now, that's fine.